src/stretchy.js

```diff
@@ -453,48 +151,86 @@ const groupLength = function(arg) {
 const svgSpan = function(group, options) {
     // Create a span with inline SVG for the element.
     const label = group.value.label.substr(1);
-    let height = 0;
-    let depth = 0;
-    let imageName = "";
+    let height;
+    let viewBoxWidth = 400000;  // default
+    let viewBoxHeight;
+    let width;
     let minWidth = 0;
+    let path;
+    let pathName;
+    let align;
+    let svgNode;
 
     if (utils.contains(["widehat", "widetilde", "undertilde"], label)) {
         // There are four SVG images available for each function.
         // Choose a taller image when there are more characters.
         const numChars = groupLength(group.value.base);
+
         if (numChars > 5) {
-            height = 0.312;
-            imageName = (label === "widehat" ? "widehat" : "tilde") + "4";
+            viewBoxHeight = (label === "widehat" ? 420 : 312);
+            viewBoxWidth = (label === "widehat" ? 2364 : 2340);
+            // Next get the span height, in 1000 ems
+            height = (label === "widehat" ? 0.42 : 0.34);
+            pathName = (label === "widehat" ? "widehat" : "tilde") + "4";
         } else {
             const imgIndex = [1, 1, 2, 2, 3, 3][numChars];
             if (label === "widehat") {
-                height = [0, 0.24, 0.30, 0.30, 0.36, 0.36][numChars];
-                imageName = "widehat" + imgIndex;
+                viewBoxWidth = [0, 1062, 2364, 2364, 2364][imgIndex];
+                viewBoxHeight = [0, 239, 300, 360, 420][imgIndex];
+                height = [0, 0.24, 0.3, 0.3, 0.36, 0.42][imgIndex];
+                pathName = "widehat" + imgIndex;
             } else {
-                height = [0, 0.26, 0.30, 0.30, 0.34, 0.34][numChars];
-                imageName = "tilde" + imgIndex;
+                viewBoxWidth = [0, 600, 1033, 2339, 2340][imgIndex];
+                viewBoxHeight = [0, 260, 286, 306, 312][imgIndex];
+                height = [0, 0.26, 0.286, 0.3, 0.306, 0.34][imgIndex];
+                pathName = "tilde" + imgIndex;
             }
         }
+        path = new domTree.pathNode(pathName);
+        align = "none";
+        svgNode = new domTree.svgNode([path], "100%", height,
+            null, viewBoxWidth, viewBoxHeight, align);
+
     } else {
         const imgData = katexImagesData[label];
-        height = imgData[0];
-        depth = imgData[1];
-        imageName = imgData[2];
-        minWidth = imgData[3];
+        const numSvgChildren = imgData[0].length;
```

Comment:
Instead of indexing into `imgData` for each value, using destructuring would help the reader know which value was which without having to scroll up to the definition of `katexIamgeData`, e.g.
```
const [paths, minWidth, height, align] = katexImagesData[label];
```

Reply:
Will do.